src/main/java/com/gregtechceu/gtceu/common/item/tool/behavior/ToolModeSwitchBehavior.java

```diff
@@ -64,6 +85,28 @@ public void addBehaviorNBT(@NotNull ItemStack stack, @NotNull CompoundTag tag) {
                         WrenchModeType.values()[tagCompound.getByte("Mode")].getName()), true);
             }
             return InteractionResultHolder.success(itemStack);
+        } else if (blockHitResult.getType() == HitResult.Type.BLOCK) {
+            BlockState state = world.getBlockState(blockHitResult.getBlockPos());
+            BlockPos pos = blockHitResult.getBlockPos();
+            Set<GTToolType> toolTypes = ToolHelper.getToolTypes(itemStack);
+            if (toolTypes.contains(GTToolType.WRENCH) && GTCEu.Mods.isCreateLoaded() &&
+                    AllTags.AllBlockTags.WRENCH_PICKUP.matches(state)) {
```

Comment:
Add this tag (the id is `create:wrench_pickup`) to CustomTags instead of referencing create's class.
```suggestion
            if (toolTypes.contains(GTToolType.WRENCH) && 
                    state.is(CustomTags.CREATE_WRENCH_PICKUP)) {
```